src/components/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { remove } from '../api/todos';
+import { FilterTypes } from '../types/Filter';
+import { Todo } from '../types/Todo';
+import { Filter } from './Filter';
+
+interface Props {
+  todos: Todo[],
+  tabs: FilterTypes[],
+  selectedTabId: string,
+  onTabSelected: (value: FilterTypes) => void,
+  setError: (value: string) => void,
+  setTodos: (value: Todo[]) => void,
+}
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  tabs,
+  selectedTabId,
+  onTabSelected,
+  setTodos,
+  setError,
+}) => {
+  const notCompleted = todos.filter(({ completed }) => !completed);
+  const completedTodos = todos.filter(({ completed }) => completed);
+
+  const handlerClick = (completedTodo: Todo[]) => {
+    const fetchData = async () => {
+      try {
+        completedTodo.forEach(async (todo) => {
+          await remove(todo.id);
+        });
+
+        setTodos([...notCompleted]);
+      } catch (errorFromServer) {
+        setError('Unable to delete a todo');
+      }
+    };
+
+    fetchData();
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${notCompleted.length} items left`}
+      </span>
+
+      <Filter
+        tabs={tabs}
+        selectedTabId={selectedTabId}
+        onTabSelected={onTabSelected}
+      />
+      {completedTodos.length > 0 && (
+        <button
+          data-cy="ClearCompletedButton"
+          type="button"
+          className="todoapp__clear-completed"
+          onClick={() => handlerClick(completedTodos)}
+        >
+          Clear completed
+        </button>
+      )}
```

Comment:
Make sure that it looks like on mate demo (you can check on dev tools how it works there)
<img width="557" alt="Screenshot 2022-10-05 at 10 39 31" src="https://user-images.githubusercontent.com/47606379/194006545-5b21c9be-b8e9-472e-a0c2-b7b2bfad3cbe.png">